server/conn.go

```diff
@@ -386,7 +387,8 @@ func parseHandshakeResponseBody(packet *handshakeResponse41, data []byte, offset
 			row := data[offset : offset+int(num)]
 			attrs, err := parseAttrs(row)
 			if err != nil {
-				log.Warn("parse attrs error:", errors.ErrorStack(err))
+				logutil.Logger(ctx).Warn("parse attrs error", zap.Error(err),
```

Comment:
how about `parse attrs failed`? the word "error" is somewhat duplicated with "err".